alienfile

```diff
@@ -0,0 +1,41 @@
+use alienfile;
+use Config;
+
+configure { requires 'Alien::Build::Plugin::Build::Make' => '0.01' };
+
+plugin 'Probe::Vcpkg' => (
+    name     => 'libsodium',
+    ffi_name => 'libsodium',
```

Comment:
nice Visual C++ support.  I can test this next time I'm in front of my  Windows box.